api/PerformanceObserver.json

```diff
@@ -269,7 +274,7 @@
               "version_added": false
             },
             "nodejs": {
-              "version_added": "8.5.0"
+              "version_added": "16.0.0"
```

Comment:
https://github.com/legendecas/node/commit/f3eb224c83f96cff9231f17b1c0a36b762942f48 seems to be the right commit, but 16.0.0 looks good.